projects/igniteui-angular/src/lib/services/transaction/transaction.ts

```diff
@@ -18,6 +19,7 @@ export interface Transaction {
     id: any;
     type: TransactionType;
     newValue: any;
+    cloneStrategy?: IDataCloneStrategy;
```

Comment:
we do not need `cloneStrategy` in the transaction interface. You are only using this in `DataUtils.mergeHierarchicalTransactions` function. You can add `cloneStartegy` parameter to this function and provide the strategy when the function gets called.